docs/common/yum.md

```diff
@@ -173,13 +114,10 @@ gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-OSG
 ```
 
-If you have your own mirror or
-configuration of the EPEL repository, you **MUST** verify that the OSG
-repository has a better yum priority than EPEL. Otherwise, you will have
-strange dependency resolution issues.
+!!! warning
+    if you have your own mirror or configuration of the EPEL repository, you **MUST** verify that the OSG repository has a better yum priority than EPEL. Otherwise, you will have strange dependency resolution issues.
 
-References
-==========
+!!! seealso "See Also"
+    -   [Basic use of Yum](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/YumRpmBasics)
+    -   [Best practices in using Yum](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallBestPractices)
```

Comment:
I think I like this is a separate section rather than an admonition, we should probably reserve their use for very specific pitfalls and whatnot.